spec/requests/admin/email_templates_controller_spec.rb

```diff
@@ -56,6 +56,30 @@ def original_text(key)
         template = templates.find { |t| t["id"] == "user_notifications.admin_login" }
         expect(template["can_revert"]).to eq(false)
       end
+
+      it "includes custom email template keys added via modifier" do
+        custom_keys = %w[custom.email_template_one custom.email_template_two]
+
+        # Clear the memoized email_keys before testing
+        Admin::EmailTemplatesController.instance_variable_set(:@email_keys, nil)
```

Comment:
If I'm reading the controller right, the email_keys in Core are memoized, and your modifier is applied to the memoized value and itself isn't memoized, so you shouldn't need this line or the one in the cleanup